mayastor/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
The top-level `.gitignore` already has a `.idea` entry so these shouldn't be committed.

Reply:
Oh my gosh whoops. :sweat: Sorry.